trunk/source/Library-UltimateTest/src/de/uni_freiburg/informatik/ultimate/test/UltimateTestCase.java

```diff
@@ -183,7 +184,12 @@ public void test() {
 				}
 				if (th != null) {
 					message += " (Ultimate threw an Exception: " + th.getMessage() + ")";
+					if (result == TestResult.IGNORE) {
+						skipTest(message, th);

```

Comment:
are you sure its a good idea to throw a new exception and dont keep the old one? at least as wrapped exception?

Reply:
@maul-esel wrote this 🙂

Reply:
Ok, so this was a long time ago, but: From my reading of the code, it seems like we (now) do keep the old exception `th` as a wrapped exception, right?